client/app/index.js

```diff
@@ -34,8 +34,7 @@ import 'script-loader!@ovh-ux/ovh-utils-angular/bin/ovh-utils-angular.min.js';
 import 'script-loader!@ovh-ux/ovh-utils-angular/lib/core.js';
 import 'script-loader!punycode/punycode.js';
 import 'script-loader!uri.js/src/uri.js';
-import 'script-loader!@bower_components/URIjs/src/URI.min.js';
-import 'script-loader!ipaddr.js/ipaddr.min.js';
+import 'script-loader!URIjs/src/URI.min.js';
```

Comment:
Is there any difference with L36?

Reply:
https://github.com/nathggns/uri.js && https://github.com/medialize/URI.js 😕